src/DropdownMenu.js

```diff
@@ -53,7 +55,10 @@ const DropdownMenu = (props, context) => {
     const position2 = right ? 'end' : 'start';
     attrs.placement = `${position1}-${position2}`;
     attrs.component = tag;
-    attrs.modifiers = !flip ? noFlipModifier : undefined;
+    attrs.modifiers = !flip ? {
+      ...noFlipModifier,
+      ...modifiers,
```

Comment:
If we want no-flip ... should we reverse the order here? That way if one specifies `flip={false} modifiers={{flip:true}}` flip will win out on the modifiers?  Rare case, just trying to think which should win?